### PR TITLE
completed

### PR DESCRIPTION
**Pull request by luke-dowling**

### CODE REVIEW

File: luke/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Q1.In one night, a movie theater sells tickets for 6450 dollars.Each ticket costs 15 dollars.How many tickets were sold ?
+let tiketsSold = 6450 / 15;
+console.log(tiketsSold);
+
+// Q2.Sylvia's income is 500 dollars per week. How much does Sylvia makes every year?
+let yearIncome = 500 * 12;
```
Comment (ciamac-da) [inline luke/index.js]: Her salary is 500 dollars per week and not month, therefore you have to multiply 500 with 52(weeks per year)

File: luke/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+console.log("$" + yearIncome);
+
+// Q3.Calculate the percentage of 17 / 30. Expected output: number %
+let percentage = (17 / 30) * 100;
```
Comment (ciamac-da) [inline luke/index.js:10]: Good Job. Also there is another alternative to solve this question: 
let percentage = (17*100) / 30

**Review by ciamac-da — CHANGES_REQUESTED** [review]
hey Luke good job. You just had one mistake about  the second question! Please read my comment and fix the code again. Thanks:)

File: luke/index.js (hunk 1 above)
Comment (luke-dowling) [reply luke/index.js]: Hey yeup just made that change will push it soon :)

File: luke/index.js (hunk 2 above)
Comment (luke-dowling) [reply luke/index.js:10]: cool ill put both in :)

**Review by ciamac-da — APPROVED** [review]

File: luke/index.js (hunk 1 above)
Comment (ciamac-da) [reply luke/index.js]: Thanks:)

**Review by ciamac-da — APPROVED** [review]

**Pull request merged**